projects/plugins/jetpack/json-endpoints/class.wpcom-json-api-site-settings-endpoint.php

```diff
@@ -1115,6 +1116,13 @@ function ( $category_id ) {
 
 					break;
 
+				case 'jetpack_header_code':
+					if ( update_option( $key, $value ) ) {
```

Comment:
Should we perform any safety checks on save? Are we good with any input without any verification?

Reply:
I _think_ so, but happy to be corrected if I'm missing anything obvious. It also seems consistent with what the other options do.